app/views/courses/salary.html.erb

```diff
@@ -4,6 +4,26 @@
   <%= govuk_back_link_to(provider_course_path(course.provider.provider_code, course.course_code)) %>
 <% end %>
 
+<% if params[:copy_from].present? %>
+  <div class="govuk-warning-summary" aria-labelledby="warning-summary-heading" tabindex="-1" data-module="error-summary" data-copy-course="warning" role="alert">
```

Comment:
data-module="error-summary"

?

Reply:
Looks odd but it means we can re-use the functionality from this module out of govuk-frontend https://github.com/alphagov/govuk-frontend/blob/master/src/components/error-summary/error-summary.js